Tests/Command/RouterDebugExposedCommandTest.php

```diff
@@ -49,9 +49,9 @@ public function testExecute()
         $tester = new CommandTester($command);
         $tester->execute(array());
 
-        $this->assertRegExp('/literal(.*ANY){3}.*\/literal/', $tester->getDisplay());
-        $this->assertRegExp('/blog_post(.*ANY){3}.*\/blog-post\/{slug}/', $tester->getDisplay());
-        $this->assertRegExp('/list(.*ANY){3}.*\/literal/', $tester->getDisplay());
+        $this->assertMatchesRegularExpression('/literal(.*ANY){3}.*\/literal/', $tester->getDisplay());
+        $this->assertMatchesRegularExpression('/blog_post(.*ANY){3}.*\/blog-post\/{slug}/', $tester->getDisplay());
+        $this->assertMatchesRegularExpression('/list(.*ANY){3}.*\/literal/', $tester->getDisplay());
```

Comment:
💡 This is necessary for PHPUnit 9 compatibility. We need PHPUnit 9 because earlier versions are not compatible with PHP 8. Symfony's PhpUnitBridge provides a polyfill for those methods.